javascript/testing_javascript/more_testing.md

```diff
@@ -64,7 +64,7 @@ If we had written this program with TDD it is very likely that it would have loo
 ### Assignment 
 
 <div class="lesson-content__panel" markdown="1">
-1. If you haven't already, watch the 'mocking' videos from [this series](https://www.youtube.com/watch?v=3PjdxjWK0F0).
+1. Check out the mocking portion from [this crash course](https://www.youtube.com/watch?v=ajiAl5UNzBU&ab_channel=LaithAcademy).
```

Comment:
```suggestion
1. Check out the mocking portion from [Jest Crash Course](https://www.youtube.com/watch?v=ajiAl5UNzBU&ab_channel=LaithAcademy).
```

Another tweak to the link text. Out of context "this crash course" may be confusing.